### PR TITLE
fix: validator item opens two tabs

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: https://github.com/lightdash/lightdash/issues/6981

### Description:

removes unnecessary anchor inside first td

### CODE REVIEW

File: packages/frontend/src/components/SettingsValidator/ValidatorTable/index.tsx [hunk 1]
```diff
@@ -107,63 +106,47 @@ const TableValidationItem = forwardRef<
...
-                <Anchor
-                    sx={{
-                        color: 'unset',
-                        ':hover': {
-                            color: 'unset',
-                            textDecoration: 'none',
-                        },
-                    }}
-                    href={getLinkToResource(validationError, projectUuid)}
-                    target="_blank"
```
Comment (joaoviana) [inline packages/frontend/src/components/SettingsValidator/ValidatorTable/index.tsx:119]: I see the benefit in keeping this `Anchor`, so users can copy the row link.  Could we instead do something with the click event in `handleOnValidationErrorClick`? 

Or remove the click event and keep the `Anchor`?

**Review by joaoviana — CHANGES_REQUESTED** [review]
one q on UX

File: packages/frontend/src/components/SettingsValidator/ValidatorTable/index.tsx (hunk 1 above)
Comment (IrakliJani) [reply packages/frontend/src/components/SettingsValidator/ValidatorTable/index.tsx]: @joaoviana updated PR and preserved old behavior. its not possible to semantically wrap table with anchor without doing some hacks so this should be good enough 👍

File: packages/frontend/src/components/SettingsValidator/ValidatorTable/index.tsx [hunk 2]
```diff
@@ -108,15 +108,18 @@ const TableValidationItem = forwardRef<
...
                         ':hover': {
                             color: 'unset',
                             textDecoration: 'none',
                         },
                     }}
-                    href={getLinkToResource(validationError, projectUuid)}
-                    target="_blank"
+                    onClick={(e) => {
+                        e.stopPropagation();
+                    }}
```
Comment (joaoviana) [inline packages/frontend/src/components/SettingsValidator/ValidatorTable/index.tsx:122]: that's the one!

**Review by joaoviana — APPROVED** [review]
thank you, merge!

**Pull request merged**